src/views/editor/mod.rs

```diff
@@ -1113,6 +1104,8 @@ impl TextLayoutProvider for Editor {
 
             let mut attrs = attrs;
             if let Some(fg) = phantom.fg {
+                //TODO: Phantom text color?
+                // attrs = self.editor_style ;
```

Comment:
Phantom text can have individually different colors in Lapce as its used for inlay hints, information, warnings, and errors.
Could have it default to the phantom color.